### PR TITLE
obsolete MONDO:0005409 variant Creutzfeld Jacob disease. Closes #847

### LINKED ISSUE #847: Merge and obsolete one of two vCJD terms
**Mondo term (ID and Label)**
variant Creutzfeld Jacob disease (MONDO:0005409)

**Reason for deprecation**
Appears to be a spelling variation for:
variant Creutzfeldt-Jakob disease (MONDO:0007012) which has more xrefs associated with it.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -123769,13 +123767,16 @@ xref: ICD10:A81.01 {source="EFO:1001233", source="MONDO:equivalentTo"}
 xref: ICD9:046.11 {source="EFO:1001233", source="MONDO:equivalentTo", source="i2s"}
 xref: MedDRA:10064199 {source="EFO:1001233"}
 xref: MESH:D016643 {source="EFO:1001233", source="DOID:5435", source="MONDO:directSiblingOf"}
+xref: MONDO:0005409 {source="MONDO:obsoleteEquivalent"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: did the script introduce this? We should fix

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: I'm not sure I understand what part you think the script introduced or what needs to be fixed?

Comment (cmungall) [reply src/ontology/mondo-edit.obo]: look on gh not in mail client. mondo shouldn't xref itself. Don't worry about it now but we should implement a check

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: I thought we were meant to xref the term that was obsoleted, this is in the editors instructions (https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/), see this example:

image

Is that what you meant? Or I am still missing something?

**Pull request merged**